tests/snapshots/run_single_files__linearize_match.hvm.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_single_files/linearize_match.hvm
+---
+λa match a { 0: λb b; +: λd (+ d c) }
```

Comment:
This test result is incorrect, `c` is a free variable.

Reply:
Same as the above